src/librustc_borrowck/diagnostics.rs

```diff
@@ -198,7 +198,7 @@ fn main() {
 ```
 "##,
 
-E0386: r##"
+/*E0386: r##"
```

Comment:
@nagisa are you saying that this modification (the commenting out here) is okay as is? Or are you saying it shouldn't stay here under `register_long_diagnostics!`, and should ... move to `register_diagnosics!`...?

I think you must mean the former, that this is okay as is... Let us know if I misunderstand.